core/App/theme.ts

```diff
@@ -116,6 +119,9 @@ const BrandColors: BrandColors = {
   highlight: '#FCBA19',
   primaryBackground: '#000000',
   secondaryBackground: '#313132',
+  modalPrimaryBackground: '#000000',
+  modalSecondaryBackground: '#313132',
+  critical: '#42803E',
```

Comment:
I don't think this makes sense as a 'brand' colour. If anything its only being used as a button style so we can leave it there for now.

Reply:
change completed